docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,8 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI secure boot on and off when provisioning a host.
+
+why we need it?
```

Comment:
This is probably going to be formatted as a wall of text, not the way you expect